Test failures with Golang 1.16 beta1

With Docker 1.16-beta1, some tests in package `github.com/open-policy-agent/opa/tester` are failing:

```
=== RUN   TestPrettyReporterVerboseBenchmark
    reporter_test.go:380: Expected:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123 ns/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
        
        
        Got:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123.0 ns/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
--- FAIL: TestPrettyReporterVerboseBenchmark (0.00s)
=== RUN   TestPrettyReporterVerboseBenchmarkShowAllocations
    reporter_test.go:457: Expected:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
        
        
        Got:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123.0 ns/op	       123.0 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
--- FAIL: TestPrettyReporterVerboseBenchmarkShowAllocations (0.00s)
=== RUN   TestPrettyReporterVerboseBenchmarkShowAllocationsGoBenchFormat
    reporter_test.go:535: Expected:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        BenchmarkDataFooBarTestBaz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
        
        
        Got:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        BenchmarkDataFooBarTestBaz	    1000	       123.0 ns/op	       123.0 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
--- FAIL: TestPrettyReporterVerboseBenchmarkShowAllocationsGoBenchFormat (0.00s)
```

The failure is due to a formatting change: the time per operation is displayed `123 ns/op` in Go 1.15, and `123.0 ns/op` in Go 1.16-beta1.

--- a/tester/fixture_go1.14_test.go
+++ b/tester/fixture_go1.14_test.go
@@ -1,0 +1,59 @@
+// +build !go1.16
+
+// The outputs here change with Go 1.16, see
+// https://github.com/golang/go/commit/afe7c8d0b25f26f0abd749ca52c7e1e7dfdee8cb
+// But since we've been using 1.14 by the time these split-off fixtures
+// were introduced, the filename mentions go1.14.
+
+package tester
+
+const fixtureReporterVerboseBenchmark = `FAILURES
+--------------------------------------------------------------------------------
+data.foo.bar.test_corge: FAIL (0s)
+
+
+SUMMARY
+--------------------------------------------------------------------------------
+data.foo.bar.test_baz	    1000	       123 ns/op
+data.foo.bar.test_qux: ERROR (0s)
+  some err
+data.foo.bar.test_corge: FAIL (0s)
+--------------------------------------------------------------------------------
+PASS: 1/3
+FAIL: 1/3
+ERROR: 1/3
+`
+
+const fixtureReporterVerboseBenchmarkShowAllocations = `FAILURES
+--------------------------------------------------------------------------------
+data.foo.bar.test_corge: FAIL (0s)
+
+
+SUMMARY
+--------------------------------------------------------------------------------
+data.foo.bar.test_baz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
+data.foo.bar.test_qux: ERROR (0s)
+  some err
+data.foo.bar.test_corge: FAIL (0s)
+--------------------------------------------------------------------------------
+PASS: 1/3
+FAIL: 1/3
+ERROR: 1/3
+`
+
+const fixtureReporterVerboseBenchmarkShowAllocationsGoBenchFormat = `FAILURES
+--------------------------------------------------------------------------------
+data.foo.bar.test_corge: FAIL (0s)
+
+
+SUMMARY
+--------------------------------------------------------------------------------
+BenchmarkDataFooBarTestBaz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
+data.foo.bar.test_qux: ERROR (0s)
+  some err
+data.foo.bar.test_corge: FAIL (0s)
+--------------------------------------------------------------------------------
+PASS: 1/3
+FAIL: 1/3
+ERROR: 1/3
+`

--- a/tester/fixture_go1.16_test.go
+++ b/tester/fixture_go1.16_test.go
@@ -1,0 +1,57 @@
+// +build go1.16
+
+// See https://golang.org/cmd/go/#hdr-Build_constraints,
+// this build constraint will match go 1.16 onwards.
+
+package tester
+
+const fixtureReporterVerboseBenchmark = `FAILURES
+--------------------------------------------------------------------------------
+data.foo.bar.test_corge: FAIL (0s)
+
+
+SUMMARY
+--------------------------------------------------------------------------------
+data.foo.bar.test_baz	    1000	       123.0 ns/op
+data.foo.bar.test_qux: ERROR (0s)
+  some err
+data.foo.bar.test_corge: FAIL (0s)
+--------------------------------------------------------------------------------
+PASS: 1/3
+FAIL: 1/3
+ERROR: 1/3
+`
+
+const fixtureReporterVerboseBenchmarkShowAllocations = `FAILURES
+--------------------------------------------------------------------------------
+data.foo.bar.test_corge: FAIL (0s)
+
+
+SUMMARY
+--------------------------------------------------------------------------------
+data.foo.bar.test_baz	    1000	       123.0 ns/op	       123.0 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
+data.foo.bar.test_qux: ERROR (0s)
+  some err
+data.foo.bar.test_corge: FAIL (0s)
+--------------------------------------------------------------------------------
+PASS: 1/3
+FAIL: 1/3
+ERROR: 1/3
+`
+
+const fixtureReporterVerboseBenchmarkShowAllocationsGoBenchFormat = `FAILURES
+--------------------------------------------------------------------------------
+data.foo.bar.test_corge: FAIL (0s)
+
+
+SUMMARY
+--------------------------------------------------------------------------------
+BenchmarkDataFooBarTestBaz	    1000	       123.0 ns/op	       123.0 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
+data.foo.bar.test_qux: ERROR (0s)
+  some err
+data.foo.bar.test_corge: FAIL (0s)
+--------------------------------------------------------------------------------
+PASS: 1/3
+FAIL: 1/3
+ERROR: 1/3
+`

--- a/tester/reporter_test.go
+++ b/tester/reporter_test.go
@@ -359,23 +359,7 @@ func TestPrettyReporterVerboseBenchmark(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	exp := `FAILURES
---------------------------------------------------------------------------------
-data.foo.bar.test_corge: FAIL (0s)
-
-
-SUMMARY
---------------------------------------------------------------------------------
-data.foo.bar.test_baz	    1000	       123 ns/op
-data.foo.bar.test_qux: ERROR (0s)
-  some err
-data.foo.bar.test_corge: FAIL (0s)
---------------------------------------------------------------------------------
-PASS: 1/3
-FAIL: 1/3
-ERROR: 1/3
-`
-
+	exp := fixtureReporterVerboseBenchmark
 	if exp != buf.String() {
 		t.Fatalf("Expected:\n\n%v\n\nGot:\n\n%v", exp, buf.String())
 	}
@@ -436,23 +420,7 @@ func TestPrettyReporterVerboseBenchmarkShowAllocations(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	exp := `FAILURES
---------------------------------------------------------------------------------
-data.foo.bar.test_corge: FAIL (0s)
-
-
-SUMMARY
---------------------------------------------------------------------------------
-data.foo.bar.test_baz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
-data.foo.bar.test_qux: ERROR (0s)
-  some err
-data.foo.bar.test_corge: FAIL (0s)
---------------------------------------------------------------------------------
-PASS: 1/3
-FAIL: 1/3
-ERROR: 1/3
-`
-
+	exp := fixtureReporterVerboseBenchmarkShowAllocations
 	if exp != buf.String() {
 		t.Fatalf("Expected:\n\n%v\n\nGot:\n\n%v", exp, buf.String())
 	}
@@ -514,23 +482,7 @@ func TestPrettyReporterVerboseBenchmarkShowAllocationsGoBenchFormat(t *testing.T
 		t.Fatal(err)
 	}
 
-	exp := `FAILURES
---------------------------------------------------------------------------------
-data.foo.bar.test_corge: FAIL (0s)
-
-
-SUMMARY
---------------------------------------------------------------------------------
-BenchmarkDataFooBarTestBaz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
-data.foo.bar.test_qux: ERROR (0s)
-  some err
-data.foo.bar.test_corge: FAIL (0s)
---------------------------------------------------------------------------------
-PASS: 1/3
-FAIL: 1/3
-ERROR: 1/3
-`
-
+	exp := fixtureReporterVerboseBenchmarkShowAllocationsGoBenchFormat
 	if exp != buf.String() {
 		t.Fatalf("Expected:\n\n%v\n\nGot:\n\n%v", exp, buf.String())
 	}

